src/AutoScrum.AzureDevOps/Models/DevOpsWorkItem.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace AutoScrum.AzureDevOps.Models
+{
+    public class DevOpsWorkItem
```

Comment:
Consistent naming

Change to `AzureDevOpsWorkItem` :)

Reply:
or  just `WorkItem`

Reply:
Hmm, this becomes a pain managing what is what as it conflicts with WorkItem from Nuget.
Externally it doesn't conflict, so I guess it's ok but not ideal.

Reply:
I'll map it as WorkItemModel allias when there is a conflict.